recipes-devtools/rust/rust-version.inc

```diff
@@ -1,6 +1,6 @@
 # Note: if you adjust this, you'll also need to change the hashes in
 # rust-source.bb
 SOURCE_NAME = "rust"
-PV = "1.10.0"
+PV = "1.12.0"
```

Comment:
> I just wanted to test another branch tickling GitHub + Jenkins in the right way.

No problem, happy to see all of the recent work that has been done on the CI front and continuing to work with upstream to reduce patches that are required.  As a stopgap for supporting various stable versions of rust with a more regular cadence @nastevens and I are currently maintaining the [meta-rust-bin](https://github.com/rust-embedded/meta-rust-bin) layer which provides similar functionality but using snapshot releases for host/target from the official mirrors.

Ultimately, we believe the meta-rust layer takes the right approach but found that in the interim going another route met our immediate needs best since our target happens to be one of the ones for which rust provides host/target executables which are "good enough" for our (and perhaps most) use cases.